src/io/cli.go

```diff
@@ -0,0 +1,32 @@
+package io
+
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+)
+
+type symbol string
+
+const (
+	//Checkmark is a checkmark symbol, that can be printed to the terminal
+	Checkmark symbol = "✓"
+	//Cross is a cross symbol, that can be printed to the terminal
+	Cross symbol = "˟"
+)
+
+//PrintGreenCheckmark prints the msg prephased with a green checkmark to the stdout
+//Used to indicate that someething was a success
+func PrintGreenCheckmark(msg string) {
```

Comment:
Wondering if we should name this PrintFailure and PrintSuccess and just let what type of symbol be an implementation detail - then it would also make it easier if we were to change things to use emojis or whatever at a later point